hack/e2e-internal/e2e-up.sh

```diff
@@ -37,6 +37,7 @@ if [[ "${FEDERATION:-}" == "true" ]]; then
     # select each one and call federated-up
     for zone in ${E2E_ZONES};do
         (
+        export CLUSTER_NAME="${zone}"
```

Comment:
@madhusudancs, In case of gke, CLUSTER_NAME is set in set-federation-zone-vars [here](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-util.sh#L64), is it possible to move this export to same function?
Else LGTM